packages/safe-core-sdk/tests/threshold.test.ts

```diff
@@ -1,35 +1,57 @@
 import chai from 'chai'
 import chaiAsPromised from 'chai-as-promised'
-import { deployments, ethers } from 'hardhat'
+import { deployments, ethers, waffle } from 'hardhat'
 import EthersSafe from '../src'
+import { ContractNetworksConfig } from '../src/configuration/contracts'
+import { GnosisSafe } from '../typechain'
+import { getMultiSend, getSafeWithOwners } from './utils/setup'
 import { getAccounts } from './utils/setupConfig'
-import { getSafeWithOwners } from './utils/setupContracts'
 chai.use(chaiAsPromised)
 
+interface SetupTestsResult {
```

Comment:
This interface seems to be defined almost in each test file, maybe move it out to `utils`?